### DISCUSSION TITLE
Face detection for PI camera

### DISCUSSION THREAD

**Discussion by sadoyan** [esimov/pigo#65] [General]
Hello, 

I've used some third party rasperry pi camera readers and can capture stream with pure GO without OpenCV and python. 
I can also transfer the image as byte arrays via channel and and find faces . Seems this all works. 

dets = classifier.ClusterDetections(dets, 0.2) returns something  when I put the camera at my face. 

But the thing is that it's not clear how to use these dets, so I can put it on real image ang get jpegs with rectangles or rounds arrround face. 

Can you please please advice, or share some code snippets ? 

Thanks,

**Comment by esimov** [comment]
It's quite simple, dets returns the number of detected faces along with other information like the detection coordinates. It's up to you what you will be doing with these informations. If you can past in some code snippets or you can detail what you want to achieve I will be able to help. There is an `examples` folder, you can check it for reference. Also there is another repo https://github.com/esimov/pigo-wasm-demos which you can use for documentation.
